pkg/log/zap_logger.go

```diff
@@ -1,15 +1,21 @@
 package log
 
 import (
+	"github.com/erwanlbp/trading-bot/pkg/eventbus"
+	"github.com/erwanlbp/trading-bot/pkg/eventbus/eventdefinition"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 type Logger struct {
 	*zap.Logger
+	*eventbus.Bus
```

Comment:
Un Logger n'est pas un EventBus du coup c'est plutot
```suggestion
	EventBus *eventbus.Bus
```